src/errors/errors.ts

```diff
@@ -146,6 +146,20 @@ module Errors {
     }
   }
 
+
+  export class CarpoolRequestConflict extends Exception {
```

Comment:
Nitpick: Can you append `Exception` to these classes. Like `CarpoolRequestConflictException`?